src/test/resources/testsData.properties

```diff
@@ -1,3 +1,15 @@
 userName=konstantinliauchuk
 userPassword=11436811a
-uri=https://mail.yandex.com/
\ No newline at end of file
+mailYandexUrl=https://mail.yandex.com/
+seleniumEasySelectUrl=https://demo.seleniumeasy.com/basic-select-dropdown-demo.html
+seleniumEasyAlertUrl=https://demo.seleniumeasy.com/javascript-alert-box-demo.html
+seleniumUserUrl=https://demo.seleniumeasy.com/dynamic-data-loading-demo.html
```

Comment:
it is better to make name shorter. for example : select.dropdown.url, js.alert.url, dynamic.loading.url and etc.

Reply:
fixed